src/argilla/client/datasets.py

```diff
@@ -1079,10 +1078,11 @@ def spans2iob(example):
             .map(lambda example: {"ner_tags": spans2iob(example)})
         )
         new_features = ds.features.copy()
-        new_features["ner_tags"] = [class_tags]
+        new_features["ner_tags"] = datasets.Sequence(feature=class_tags)
         ds = ds.cast(new_features)
+        ds = ds.remove_columns(set(ds.column_names) - set(["tokens", "ner_tags"]))
```

Comment:
Not sure what would be the cost to add tests covering these new lines. but it would be nice to have them